deploy/examples/create-external-cluster-resources.py

```diff
@@ -1162,7 +1162,7 @@ def create_checkerKey(self, user):
         # check if user already exist
         user_key = self.check_user_exist(entity)
         if user_key != "":
-            return user_key, f"{entity.split('.', 1)[1]}"
```

Comment:
So this was a regression from #12941? The second return value was just not needed?